frida_tools/tracer.py

```diff
@@ -345,15 +345,39 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern (pattern) {
+    var m;
+    var f;
+    var res = pattern.split ('!');
+    
```

Comment:
Let's move these closer to where they're used:

```suggestion
    var m, f;
```